From 2ef76d203d291efbf5004484bbd5c27f697801ac Mon Sep 17 00:00:00 2001
From: Steven Rostedt <rostedt@goodmis.org>
Date: Wed, 23 Mar 2016 10:16:28 -0400
Subject: [PATCH 073/134] tools lib traceevent: Remove redundant CPU output

Commit 8533bda27f64 "tools lib traceevent: Split pevent_print_event()
into specific functionality functions" Broke apart the function
pevent_print_event() into three functions. The first function prints
the comm, pid and CPU, the second prints the timestamp. But that commit
added the printing of the CPU in the timestamp function, which now
causes pevent_print_event() to duplicate the CPU output.

Remove the redundant printing of the record's CPU from the timestamp
function.

Link: http://lkml.kernel.org/r/20160323101628.459375d2@gandalf.local.home

Fixes: 8533bda27f64 "tools lib traceevent: Split pevent_print_event() into specific functionality functions"
Signed-off-by: Steven Rostedt <rostedt@goodmis.org>
---
 event-parse.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/event-parse.c b/event-parse.c
index b12b2cd1b06b..82400eada24e 100644
--- a/event-parse.c
+++ b/event-parse.c
@@ -5453,10 +5453,8 @@ void pevent_print_event_time(struct pevent *pevent, struct trace_seq *s,
 	}
 
 	if (pevent->latency_format) {
-		trace_seq_printf(s, " %3d", record->cpu);
 		pevent_data_lat_fmt(pevent, s, record);
-	} else
-		trace_seq_printf(s, " [%03d]", record->cpu);
+	}
 
 	if (use_usec_format) {
 		if (pevent->flags & PEVENT_NSEC_OUTPUT) {
-- 
2.5.5

